config/crd/bases/carto.run_workloads.yaml

```diff
@@ -322,6 +322,27 @@ spec:
               resources:
                 description: Resource constraints for the application. See https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
                 properties:
+                  claims:
```

Comment:
I don't understand why this is turning up now?. I wonder if previous commits did not remember to generate.